api/proto/external.proto

```diff
@@ -118,7 +118,7 @@ message RootIdentity {
     string fog_report_id = 3;
     // Optional fog authority subjectPublicKeyInfo.
     // Empty when not in use.
-    bytes fog_authority_spki = 5;
+    bytes fog_authority_spki = 4;
```

Comment:
This was discussed, https://sentzglobal.slack.com/archives/C015GEV9UTT/p1746108778400469